internal/task/task/bs/start_tgtd.go

```diff
@@ -127,7 +127,7 @@ func NewStartTargetDaemonTask(curveadm *cli.CurveAdm, cc *configure.ClientConfig
 		ExecOptions: curveadm.ExecOptions(),
 	})
 	t.AddStep(&step.ContainerExec{
-		Command:     "tgtd -f &",
+		Command:     "-d tgtd -f &",
```

Comment:
Maybe modify the step of ContainerExec it better?
ContainerExec{
    Command: "tgtd -f",
    Detached: true,
    ...
}

Reply:
Oh, it's attractive! I didn't notice there was this option

Reply:
But you need to add it, now it doesn't support `--detach` option

Reply:
no problem!